components/frontend/src/report/Report.test.js

```diff
@@ -58,6 +58,7 @@ function renderReport(
                     report={reportToRender}
                     report_date={report_date}
                     settings={settings}
+                    last_update={new Date()}
```

Comment:
Can we rename thus prop to `lastUpdate`, see comment above about camelCase/snake_case.